src/odemis/util/test/filename_test.py

```diff
@@ -36,6 +36,10 @@
 # date_sl = time.strftime("%Y/%m/%d")
 # timelng_sl = time.strftime("%H/%M/%S")
 current_year = time.strftime("%Y")
+dshrtrev = time.strftime("%d%m%y")
+# Note: cannot be used in tests, as on days similar to the year, such as 20-10-20,
+# it's not clear what is the day and what is the year, but the code convention is to
+# guess DDMMYY.
 dateshrt = time.strftime("%y%m%d")
```

Comment:
This is now not used at all, so why not remove it?

Reply:
Fair enough... done.